pom.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- This is our default parent pom, it is useful to keep general versions
+	 of artifacts, plugins and maven repository definitions -->
+    <parent>
+        <groupId>com.emc.ocopea.maven</groupId>
+        <artifactId>ocopea-parent-pom</artifactId>
+        <version>0.59</version>
+    </parent>
+
+    <!-- We usually have a separate group id per repository -->
+    <groupId>com.emc.ocopea.mysql-dsb</groupId>
+    <packaging>pom</packaging>
+    <artifactId>dsb-super-parent</artifactId>
+    <name>Parent project for mysql-dsb</name>
+    <version>0.1</version>
+
+    <properties>
+        <microservice.api.version>1.0</microservice.api.version>
+        <microservice.runtime.version>0.1</microservice.runtime.version>
+        <ocopea.dsb.version>0.38</ocopea.dsb.version>
```

Comment:
lets talk about component version and how we manage them, I want it to be the same across the projects

Reply:
sure next iteration.